### PR TITLE
Small updates before publishing to NPM

### PR DESCRIPTION
**Pull request by nathan-barrett**
- Added a readme within the `docs/` folder which has documentation about how to publish to NPM and build the project
- Added TOC to readme within docs
- Removed Mozilla namespace from source file

### CODE REVIEW

**Review by alexgibson — CHANGES_REQUESTED** [review]
Spotted one small issue but otherwise this all looks great.

I don't seem to see the merge button, can you add mozmeao members to this repo like we did for traffic cop?

Edit: I also saw a couple of minor diff changes in the docs / lockfile after running `npm test`. Maybe worth running again just to make sure you have all the latest changes committed?

**Comment by alexgibson** [comment]
I also spotted we have trafficcop as a dependency here in package.json? 

```
"dependencies": {
        "@mozmeao/trafficcop": "^1.0.1"
    }
````

Shouldn't this be the other way around (i.e. once cookie helper is published to NPM, we update traffic cop to use the package as one of its dependencies)?

**Comment by nathan-barrett** [comment]
Yeah - you're right - it was the other way around and I forgot to remove it from the dependencies. Thanks!

File: src/mozilla-cookie-helper.js [hunk 1]
```diff
@@ -4,12 +4,7 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-// create namespace
-if (typeof window.Mozilla === 'undefined') {
-    window.Mozilla = {};
-}
-
-Mozilla.Cookies = {
+const CookieHelper = {
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply src/mozilla-cookie-helper.js:7]: I think this script is needed to run as far back as IE8 for stub attribution. I'm not sure that's something babel officially supports today, so this script may need to be kept as ES5, at least for now.

**Review by alexgibson — CHANGES_REQUESTED** [review]
Changes look good, but we should keep this as ES5 for now I think. Maybe we could do some testing in a separate issue later on to see if transpiling is something we could do

**Review by alexgibson — APPROVED** [review]
r+

**Pull request merged**